packages/driver/src/cy/logGroup.ts

```diff
@@ -38,5 +38,7 @@ export default (Cypress, userOptions: Cypress.LogGroup.Config, fn: Cypress.LogGr
 
   cy.queue.insert(restoreCmdIndex, endLogGroupCmd.set('fn', forwardYieldedSubject))
 
-  return fn(log)
+  fn(log)
+
+  return log
```

Comment:
I think we'll probably want to change this as we see the log group API used in more cases, but this was the least intrusive way I could think of to implement this for the sake of `cy.origin()`.

Reply:
@emilyrohrbough I made updates so this is no longer changing the log grouping api.